frontend/components/vault/WithdrawDaiFromVat.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <TextBlock v-if="isExplanationsShown">
+            After liquidating a vault, the DAI-based liquidation incentive will end up in your VAT account. One more
+            transaction is required to move that DAI to your wallet.
+        </TextBlock>
+        <div class="my-4">
+            <WalletAuthorizationCheckPanel
+                :disabled="!isWalletConnected"
+                :wallet-address="walletAddress"
+                :is-wallet-authorized="isWalletAuthorized"
+                :is-explanations-shown="isExplanationsShown"
+                :is-loading="isAuthorizing"
+                @authorizeWallet="$emit('authorizeWallet')"
+            />
+        </div>
+        <div class="flex justify-end mt-2 gap-5">
+            <BaseButton
+                :disabled="!isWalletConnected || !isWalletAuthorized || !hasDaiToWithdraw || isWithdrawing"
+                @click="$emit('manageVat')"
+            >
+                Withdraw DAI partially
```

Comment:
I would rephrase this to:

"Partially withdraw DAI", I think that sounds better.

Or do we take another description, as this opens the VAT modal right? Something in the direction of:

"Manage VAT"?